tests/aria-tooltip-name.ts

```diff
@@ -0,0 +1,58 @@
+import { fixture, html, expect } from "@open-wc/testing";
+import { Scanner } from "../src/scanner";
+import { ariaTooltipName } from "../src/rules/aria-tooltip-name";
+
+const scanner = new Scanner([ariaTooltipName]);
+
+const passes = [
+  `<div role="tooltip" id="al" aria-label="Name"></div>`,
+  `<div>
+    <div role="tooltip" id="alb" aria-labelledby="labeldiv"></div>
+    <div id="labeldiv">Hello world!</div>
+  </div>`,
+  `<div role="tooltip" id="combo" aria-label="Aria Name">Name</div>`,
+  `<div role="tooltip" id="title" title="Title"></div>`,
+];
+
+const violations = [
+  `<div role="tooltip" id="empty"></div>`,
+  `<div role="tooltip" id="alempty" aria-label=""></div>`,
+  `<div
+      role="tooltip"
+      id="albmissing"
+      aria-labelledby="nonexistent"
+    ></div>`,
+  `<div>
+    <div role="tooltip" id="albempty" aria-labelledby="emptydiv"></div>
+    <div id="emptydiv"></div>
+  </div>`,
+];
+
+describe("aria-tooltip-name", async function () {
+  for (const markup of passes) {
+    const el = await fixture(html`${markup}`);
```

Comment:
We should change all the other rules to use this pattern so that the markup is correct for each rule. Otherwise it's all rendered into memory and then wiped from the DOM after the first test runs.

Reply:
In fact, standardising this whole suite might be the way to go. A developer should then just provide the test cases.